Source/LuaBridge/detail/Userdata.h

```diff
@@ -57,7 +57,7 @@ namespace luabridge {
 class Userdata
 {
 protected:
-  void* m_p; // subclasses must set this
+  void* m_p = nullptr; // subclasses must set this
```

Comment:
This cannot be compiled with a pre-C++11 compiler (add `LUABRIDGE_NO_CXX11` definition for that).
So `m_p` initialization should be done through a constructor member initializer list instead.

One day I'll remove the pre-C++11 support as it's way too old. But currently there are separate tests for the legacy (the test binaries names end with "L").
https://github.com/vinniefalco/LuaBridge/blob/24f1877f967cc138c780ed78a23e03a490a9aec9/Tests/CMakeLists.txt#L90-L96

Reply:
i changed it to a initializer list